### PR TITLE
Add command to remove plugin assets from app's webroot.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: src/Shell/Task/AssetsTask.php [hunk 1]
```diff
@@ -157,6 +181,55 @@ protected function _process($plugins, $copy = false)
...
+     * @return void
+     */
+    protected function _remove($config)
+    {
+        if ($config['namespaced'] && !is_dir($config['destDir'])) {
+            $this->out(
+                $config['destDir'] . $config['link'] . ' does not exist',
+                1,
+                Shell::VERBOSE
+            );
```
Comment (markstory) [inline src/Shell/Task/AssetsTask.php]: You could use `$this->verbose()` or `$this->getIo()->verbose()` to have shorter lines.

File: tests/TestCase/Shell/Task/AssetsTaskTest.php [hunk 2]
```diff
@@ -204,4 +204,65 @@ public function testCopy()
...
+
+        $this->assertTrue(is_dir(WWW_ROOT . 'test_plugin'));
+
+        $this->assertTrue(is_dir(WWW_ROOT . 'company' . DS . 'test_plugin_three'));
+
+        $this->Task->remove();
+
+        $this->assertFalse(is_dir(WWW_ROOT . 'test_plugin'));
+        $this->assertFalse(is_dir(WWW_ROOT . 'company' . DS . 'test_plugin_three'));
+        $this->assertTrue(is_dir(WWW_ROOT . 'company'), 'Ensure namespace folder isn\'t removed');
```
Comment (markstory) [inline tests/TestCase/Shell/Task/AssetsTaskTest.php]: You could use [assertDirectoryExists](https://phpunit.de/manual/current/en/appendixes.assertions.html#appendixes.assertions.assertDirectoryExists) and `assertDirectoryNotExists()`

**Comment by ADmad** [comment]
@markstory Updates done as per feedback.

**Comment by ADmad** [comment]
It added `overwrite` option for `symlink()` and `copy()`.

**Pull request merged**